src/com/xmslyz/luhn/logic/Computation.java

```diff
@@ -0,0 +1,11 @@
+package com.xmslyz.luhn.logic;
+
+public interface Computation {
+
+    String compute (String input);
+    int setLenght();
+    void setArray(int lenght);
+    void multiplyWages();
+    void countTotal();
+    void getChecksum();
```

Comment:
Tu robisz błąd architektowniczy. Interfejs powinieneś użyć, żeby klasy spoza pakietu `logic` mogły uruchomić obliczenia. A te metody to detal implementacyjny jednego konkretnego sposobu implementacji, nie powinny być wywoływane spoza pakietu `logic` . Interfejs definiuje kontrakt pomiędzy użytkownikami a implementacją.